src/System.Memory/tests/System.Memory.Tests.csproj

```diff
@@ -148,6 +148,7 @@
     <Compile Include="ReadOnlySpan\LastIndexOf.T.cs" />
     <Compile Include="ReadOnlySpan\LastIndexOfSequence.byte.cs" />
     <Compile Include="ReadOnlySpan\LastIndexOfSequence.T.cs" />
+    <Compile Include="ReadOnlySpan\ToUpperLower.cs" />
```

Comment:
nit: sort this alphabetically, so move it down to after `ReadOnlySpan\ToString.cs`).

Reply:
mmm...seem that I've my personal alphabet or I need an holiday 😄